cv32e40x/tests/cfg/no_pulp.yaml

```diff
@@ -1,10 +0,0 @@
-name: no_pulp
```

Comment:
You've deleted all the content, but not the file.  I understand why the file is not needed for e40x, so why not delete it outright?